aiopyrq/unique_queues.py

```diff
@@ -134,6 +138,26 @@ async def ack_items(self, items: list) -> None:
         await pipeline.execute()
         await self._wait_for_synced_slaves()
 
+    async def can_rollback_item(self, item) -> bool:
+        """
+        Check if `item` has not been rolledback too many times
+
+        :param item: Anything that is convertible to str
+        """
+
+        if not self.options['max_retry'] or not self.options['max_timeout']:
```

Comment:
mozna bych specifikoval pro co jsou ty max hodnoty urcene ? Jen kdyby se tam treba pridaval do options i jiny max retry pro neco jineho? Ale to je jen na zamysleni ....